src/components/ActionBar/src/ActionBarButton.vue

```diff
@@ -317,17 +337,32 @@ export default {
 }
 
 .align_stack .InformationText {
-	font-size: 12px;
+	width: max-content;
+}
+
+.hasMainAndLabelText {
+	text-align: right;
+
+	& .InformationText {
+		text-align: left;
+	}
 }
 
 .TruncateText {
+	display: -webkit-box;
```

Comment:
I would put a comment here like "despite the -webkit prefix this style is actually supported by all modern browsers" because -webkit usually only means Safari (_and maybe_ Chrome & Opera) support it but in this case every browser supports this style, including Firefox and Edge. To save ourselves and others potential confusion later hopefully.